SPECS/k3s/k3s.spec

```
@@ -1,21 +1,21 @@
 Summary:        Lightweight Kubernetes
 Name:           k3s
-Version:        1.25.0
+Version:        1.25.5
 Release:        7%{?dist}
```

We generally bump the release if we do anything with the spec.
In this case, since we are bumping the version, we start from 1.
```suggestion
Release:        1%{?dist}
```

---

Done